### PR TITLE
hotFix/123-homeDocument new field docType

### PR DESCRIPTION
**Pull request by FelixoGudiel**
Frontend has communicated that the current homeDocument is not up to date with the requirements. This class is missing an attribute docType that should be a clasifier between documents. The values of this field should be DOCS_INSTITUCIONALES, MEMORIAS_ANUALES,MEMORIAS_ECONOMICAS,BALANCE_CUENTAS,OTROS_DOCS.

### CODE REVIEW

File: populate.json [hunk 1]
```diff
@@ -382,7 +382,8 @@
         "model": "nexong.homeDocument",
         "pk": 1,
         "fields": {
-            "title": "documentodecasa",
+            "title": "documentoDeCasa",
```
Comment (auroranavas) [inline populate.json:385]: Could you change this title to something more descriptive? Like `balance_cuentas_2023`.

**Review by auroranavas — CHANGES_REQUESTED** [review]

**Comment by FelixoGudiel** [comment]
Done!

**Review by auroranavas — COMMENTED** [review]
Now everything's perfect. I want to ask you to add two small changes to our DB. The reason is that when I was checking our deployed app I noticed two strange things that are the backend's fault. 

- The Meeting 'time' field should be `TimeField` and currently it is a DateTimeField.
- Lesson has start_date and end_date de lesson and they should be DateField.

**Review by Ahydul — APPROVED** [review]
It works correctly and the changes are reflected in the db

**Review by auroranavas — APPROVED** [review]
Thank you for going out of your way to make the project better! Good work.

**Review by manortbla — APPROVED** [review]
Thanks for the changes. Good job!

**Pull request merged**